src/site/stages/build/process-cms-exports/transformers/node-person_profile.js

```diff
@@ -14,7 +14,10 @@ const transform = (entity, { ancestors }) => ({
   )}`,
   entityMetatags: createMetaTagArray(entity.metatag.value),
   entityPublished: isPublished(getDrupalValue(entity.status)),
-  fieldBody: getDrupalValue(entity.fieldBody),
+  fieldBody:
```

Comment:
This looks good. The only other thing needed for this change is to update the output schema for this transformer. Specifically the `fieldBody` field.